File: bubble_sort.java
public class bubble_sort {

    public static void bubblesort(int[] a) {
        int n = a.length;

        for(int i = 0; i < n - 1; i++) {

            boolean flag = false;

            for(int j = 0; j < n - i - 1 ; j++) {
                if (a[j] > a[j + 1]) {
                    int temp = a[j];
                    a[j] = a[j + 1];
                    a[j + 1] = temp;
                    flag = true;
                }
            }
            if(flag == false){
                return;
            }
        }

    }
    public static void main(String[] args) {
        int[] a= { 1,4,7,8,6,25,2};
        bubblesort(a);
        for(int i = 0; i < a.length; i ++){
            System.out.print(a[i] + " ");
        }
    }
}

File: bubble_sort.js
function bubble_sort(arr) {
    for (let i=0 ; i<arr.length-1 ; i++) {
        let flag = false;
        for (let j=0 ; j<arr.length-i-1 ; j++) {
            if (arr[j] > arr[j + 1]) {
                [arr[j],arr[j+1]] = [arr[j+1],arr[j]];
                flag = true;
            }
        }
        if (!flag) {
            return;
        }
    }
}

let a = [4,7,8,9,54,6,75];
bubble_sort(a);
for (let i = 0; i < a.length; i++) {
    console.log(a[i]);
}